Automattic/jetpack
# Add new external media endpoint

Fixes #40038, endpoint part

## Proposed changes:

* Add new Google Picker WPCOM endpoints to Jetpack

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* See [Jurassic Tube instructions](https://github.com/Automattic/jetpack/blob/trunk/docs/quick-start.md#setting-up-jurassic-tube) here to setup the environment.
* Enable `Keyring_Service_Google_Photos::IS_PICKER_ENABLED`
* You can manually try the endpoints `https://_ID_.jurassic.tube/wp-json/wpcom/v2/external-media/list/google_photos` or use the changes found on https://github.com/Automattic/jetpack/pull/40087 to check them.

The flow should be the same as of WoA sites with features on.
![image](https://github.com/user-attachments/assets/08cf282b-1861-455d-8877-644bdf39da7c)
